submit/filters/tex_filters.py

```diff
@@ -241,24 +249,35 @@ def compilation_log_display(autotex_log: str, submission_id: int,
         ['ignore', 'warning', 'first'],
         ['ignore', 'undefined', 'first'],
     ]
-    abort_markup = False
+
+    # Try to create summary containing errors deemed important for user
+    # to address.
     error_summary = ''
+
+    # Keep track of any errors we've already added to error_summary
+    abort_markup = False
+    xetex_luatex_abort = False
+    emergency_stop = False
+    missing_file = False
     missing_font_markup = False
     rerun_markup = False
```

Comment:
Consider renaming some of these for extra clarity, per [decision on predicates](https://confluence.cornell.edu/pages/viewpage.action?pageId=372900900).

Reply:
I will rename some of these predicates for clarity. This code was actually written prior to the 'decision on predicates'  decision but makes sense to update at this time.